### ISSUE TITLE
Metadata required for freshwater data

### ISSUE THREAD

**Issue by CecSve** [gbif/doc-freshwater-data-publishing-guide#21]
Feedback for: https://docs.gbif-uat.org/freshwater-data-publishing-guide/en/#required-freshwater-metadata

>  Some of this information can be reported within the GBIF metadata, while other fields may be better associated with the occurrence or sampling-event data.

For clarity, maybe rephrase this sentence to:

>  Some of this information can be reported within the resource metadata, while other fields may be better associated with the occurrence or sampling-event data.

In the following paragraph, please include DwC relevant terms and link to the proposed terms that could eventually capture the data:

> Habitat **([dwc:habitat](https://dwc.tdwg.org/terms/#dwciri:habitat))** descriptions should at minimum include the [realm](https://docs.gbif-uat.org/freshwater-data-publishing-guide/en/#realm) and [biome](https://docs.gbif-uat.org/freshwater-data-publishing-guide/en/#biome) to indicate whether observations were made in freshwater and in what water body type **([dwc:waterBody](https://dwc.tdwg.org/terms/#dwc:waterBody))**

+ reference to the the [proposed terms](https://docs.gbif-uat.org/freshwater-data-publishing-guide/en/#freshwater-dwc-extension-terms) for realm(not currently proposed, but shouldn't it be?) and biome.

Generally, the [whole section](https://docs.gbif-uat.org/freshwater-data-publishing-guide/en/#required-freshwater-metadata) could use concrete fields that could be used or more links to other places in the document where reader can see specific requirements. The text reads much like the introduction on why it is important to include such data (maybe no need to repeat this?) and could benefit for more (links to) 'tools' that the publisher can use.

**Comment by jenlento** [comment]
Thank you for the suggested re-wording. We have made the change.

With respect to additional links and fields, this section is still part of the introduction, and we feel that it's best to not add too many additional terms and links to this section that might make it difficult for the reader to follow the main ideas. The details about fields to include are provided in sections 2 and 3.
